src/_variables.scss

```diff
@@ -20,3 +20,9 @@ $background: map-get($theme, background);
 // bars and our bottom bars.
 $menu-bar-height: 64px;
 $menu-bar-padding: 16px;
+
+// Some breakpoints for media queries.
+$tablet-portrait-min-width: 600px;
+$tablet-landscape-min-width: 900px;
```

Comment:
Suggestion: use the official Material design breakpoints: https://material.io/design/layout/responsive-layout-grid.html#breakpoints

This code may be useful (the strings are all css `@media` queries):

https://github.com/angular/components/blob/bb2bb9c3065ec1a7a845de072305893a7378383a/src/cdk/layout/breakpoints.ts#L10-L31

Reply:
Ah, sweet, thank you! Those material breakpoints are super useful. Will do!